### PR TITLE
[enhancement](broker-load) support compress type for old broker load, and split compress type from file format

### PR DESCRIPTION
**Pull request by Hastyshell**
## Proposed changes

As title.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/common/util/Util.java [hunk 1]
```diff
@@ -534,29 +536,38 @@ public static String bytesToHex(byte[] bytes) {
...
+        } else if (lowerFileFormat.equals("json")) {
+            return TFileFormatType.FORMAT_JSON;
+            // csv/csv_with_name/csv_with_names_and_types treat as csv format
+        } else if (lowerFileFormat.equals(FeConstants.csv) || lowerFileFormat.equals(FeConstants.csv_with_names)
+                || lowerFileFormat.equals(FeConstants.csv_with_names_and_types)
+                // TODO: Add TEXTFILE to TFileFormatType to Support hive text file format.
+                || lowerFileFormat.equals(FeConstants.text)) {
+            return TFileFormatType.FORMAT_CSV_PLAIN;
+        } else {
+            return TFileFormatType.FORMAT_UNKNOWN;
```
Comment (CalvinKirs) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/Util.java:567]: Considering maintaining the mapping relationship will be more maintainable and clear than if + magic number

**Review by CalvinKirs — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/common/util/Util.java (hunk 1 above)
Comment (TangSiyang2001) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/Util.java:567]: ok I‘ll mod it

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**